src/components/book/Book.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+
+function Book(props) {
+  const { title, author } = props;
+  return (
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you use `Arrow function` on your `Book` component for best practices